lib/modules/forgot_password/components/reset_successfully_notification.dart

```diff
@@ -0,0 +1,46 @@
+part of '../forgot_password_module.dart';
+
+class ResetSuccessfullyNotification extends StatelessWidget {
+  const ResetSuccessfullyNotification({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: const Color(0xff252525).withOpacity(0.6),
```

Comment:
a common practice for these custom color is to extract some sort of constant for them. Something like the standard `Colors` class:
```
Colors {
static const neroGray = const Color(0xff252525)
static const neroGray60 = const Color(0xff252525).withOpacity(0.6)

static const pinkyPink = const Color (0xff281921)
}

usage: Colors.neroGray60
``` 

This way you can reuse it, and the naming will be more human-readable.